README.md

```diff
@@ -1 +1,43 @@
-uProxy Windows Installer
+# uProxy Windows Installer
+
+uProxy is a browser plugin that allows users to reach the open internet through a trusted friend or uProxy cloud server.
+
+The uProxy Windows Installer creates a modified version of the Firefox Windows Installer with the uProxy extension enabled by default. This allows Windows users to get up and running with uProxy quickly.
+
+###Dev setup:
+- Need to use a Windows OS
+- [7-zip](http://www.7-zip.org/) and [signtool](https://www.microsoft.com/en-us/download/details.aspx?id=8279), which is part of Windows SDK 7.1, need to be installed
```

Comment:
Maybe move the link to cover Windows SDK 7.1, instead of signtool.  I first saw this and thought that the link was for a description of signtool.